<!DOCTYPE html>
<html lang="en">
  <head>
    <title>SSB2020 - Workshops & Forums</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

    <link href="https://fonts.googleapis.com/css?family=Amatic+SC:400,700|Work+Sans:300,400,700" rel="stylesheet">
    <link rel="stylesheet" href="fonts/icomoon/style.css">

    <link rel="stylesheet" href="css/bootstrap.min.css">
    <link rel="stylesheet" href="css/magnific-popup.css">
    <link rel="stylesheet" href="css/jquery-ui.css">
    <link rel="stylesheet" href="css/owl.carousel.min.css">
    <link rel="stylesheet" href="css/owl.theme.default.min.css">
    <link rel="stylesheet" href="css/bootstrap-datepicker.css">
    <link rel="stylesheet" href="css/animate.css">
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mediaelement@4.2.7/build/mediaelementplayer.min.css">
    
    
    
    <link rel="stylesheet" href="fonts/flaticon/font/flaticon.css">
  
    <link rel="stylesheet" href="css/aos.css">

    <link rel="stylesheet" href="css/style.css">
    
  </head>
  <body>
  
  <div class="site-wrap">

    <div class="site-mobile-menu">
      <div class="site-mobile-menu-header">
        <div class="site-mobile-menu-close mt-3">
          <span class="icon-close2 js-menu-toggle"></span>
        </div>
      </div>
      <div class="site-mobile-menu-body"></div>
    </div> <!-- .site-mobile-menu -->
    
    
    <div class="site-navbar-wrap js-site-navbar bg-white">
      
      <div class="container">
        <div class="site-navbar bg-light">
          <div class="py-1">
            <div class="row align-items-center">
              <div class="col-2">
              </div>
              <div class="col-10">
                <nav class="site-navigation text-right" role="navigation">
                  <div class="container">
                    <!-- d-lg-none -->
                    <div class="d-inline-block  ml-md-0 mr-auto py-3"><a href="#" class="site-menu-toggle js-menu-toggle"><span class="icon-menu h3"></span></a></div>
                     <!-- d-lg-block -->
                    <ul class="site-menu js-clone-nav d-none">
                      <li><a href="index.html">Home</a></li>
                      <li><a href="registration.html">Registration & Abstracts</a></li>
					  <li><a href="schedule.html">Schedule</a></li>
                      <li><a href="location.html">Location</a></li>
                      <li><a href="travel_accomodation.html">Travel & Accommodations</a></li>
                      <li><a href="workshop.html">Workshops & Forums</a></li>
					  <li><a href="activites.html">Activities</a></li>
                      <li><a href="contact.html">Contact</a></li>
					  <li><a href="acknowledgments.html">Acknowledgments</a></li>
                    </ul>
                  </div>
                </nav>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>  
  
    
    
	
    <div class="site-blocks-cover overlay" style="background-image: url(images/MWalters_5.jpg);" data-aos="fade" data-stellar-background-ratio="0.5">
      <div class="container">
        <div class="row align-items-center justify-content-center">
          <div class="col-md-10" data-aos="fade">
		  <img src="images/ssb2020_logo.jpg" alt="SSB2020" class="img-logo">

            <h1 class="mb-4 text-center" style="position:relative; top:-100px">Workshops & Forums</h1>
            </p>
          </div>
        </div>
      </div>
    </div>  


    
    <div class="site-section">
      <div class="container">
        
        <div class="row">
          <div class="mx-auto text-center mb-5 section-heading">
            <h2 class="mb-5 text-uppercase">Workshops</span></h2>
			<p style = "font-size:1.5em"> <a href="Schedule_PDF.pdf">PDF of Schedule Here</a></p>
          </div>        
         </div>
	
           <div class="text-left">
			  <h2 class="heading mb-0"><b>Friday Morning (9am-12pm):</b></h2>
			  </br>
              <h3 class="heading mb-0">How to Navigate in Tree Space</h3>
              <p><i>Primary organizer:</i> Dr. Nico Cellinese, Florida Museum of Natural History</br></br>
                 <i>Content:</i> Integrating biological data with phylogenies has become a challenge,
				 especially in view of the growing need for very large synthetic analyses.
				 This is primarily due to the arbitrary approach we take to associate traditional names to tree nodes.
				 In this workshop we will: 1) briefly introduce the rationale for generating clade names;
				 2) learn about the minimum requirements needed to generate the various types of phylogenetic definitions;
				 and 3) learn how to generate a phyloreference (machine readable taxon concept).
			  </p>
            </div>

			<br/>

            <div class="text-left">
              <h3 class="heading mb-0">Hands-On Learning with RevBayes</h3>
              <p><i>Primary organizer:</i> Dr. April Wright, Southeastern Louisiana University</br></br>
                 <i>Content:</i> This workshop will focus on using the phylogenetic estimation software RevBayes
				 in an instructional setting. We will first introduce the graphical model framework used by the software.
				 Graphical models can be used to introduce the fundamentals of probability, while also enabling transparent
				 and flexible assembly of new phylogenetic models. Then, we will discuss using RevBayes in hands-on exercises
				 for systematics and phylogenetics courses. Topics will include making use of the robust RevBayes tutorial
				 library, tailoring pre-existing tutorials to your course, contributing your tutorials to the tutorial bank,
				 using interactive computing notebooks, and integrating the software with R and Python.
			  </p>
            </div
			</br>
			</br>

            <div class="text-left">
			  <h2 class="heading mb-0"><b>Friday afternoon (1:30-5pm):</b></h2>
			  </br>
              <h3 class="heading mb-0">Using the Open Tree of Life for your Research</h3>
                <p><i>Primary organizer:</i> Dr. Emily Jane McTavish, University of California (UC) Merced</br></br>
                 <i>Content:</i> The <a href="https://opentreeoflife.github.io" target="_blank">Open Tree of Life</a> project is a phylogenetic resource
				 that offers a synthetic tree summarizing evolutionary relationships across 2.6 million taxa. OpenTree
				 (as of September 2019) also provides access to 4,168 published studies comprising 9,367 trees, and the
				 database is growing weekly. The tips of these trees are mapped to taxonomic identifiers that link across
				 taxonomic and informatics databases such as NCBI and The Global Biodiversity Information Facility (GBIF).
				 The website also provides user-friendly resources for bulk reconciliation of taxonomic names to these standardized
				 identifiers. Together these resources offer a formidable suite of tools and data for taxonomic name reconciliation
				 and phylogenetic comparisons and analyses.</br>
				 </br>
				 This workshop will cover how to understand, access and apply these data resources from the Open Tree of Life project
				 to your research questions. Topics will include accessing phylogenetic relationships for arbitrary sets of
				 taxa, applying date estimates to these trees, linking phylogenetic and geographic information, assessing
				 conflict between phylogenetic estimates, and incorporating your own phylogenetic estimates into the OpenTree of Life project.</br>
				 </br>
				 The workshop will include interactive browser based exercises, demonstrations of how to access OpenTree
				 data through the API, and tutorials in R and Python. Experience using a command line interface, R or Python
				 will be helpful, but is not required.

			  </p>
            </div>
			
		  </br>


          <div class="text-left">
              <h3 class="heading mb-0">State-dependent Diversification Models via Graphical Models and RevBayes</h3>
              <p><i>Primary organizer:</i> Dr. Rosana Zenil-Ferguson, University of Hawaii Manoa</br></br>
                 <i>Content:</i> SSE models such as BiSSE and MuSSE are powerful approaches for testing the
				 association of a character with diversification rate heterogeneity. However, they have been
				 shown to be prone to falsely identifying a positive association when diversification rate shifts
				 are correlated with a character not included in the model. One approach to reduce the possibility
				 of falsely associating a character with diversification rate heterogeneity is to incorporate a second,
				 unobserved character into the model (i.e., a Hidden State-Dependent Speciation and Extinction (HiSSE) model.
				 This workshop will discuss hidden state models and their uses in evolutionary biology, and participants
				 will use RevBayes to set up such a model and use it to test some basic questions about trait associations.
			  </p>
            </div>

            <div class="text-left">
			  <h2 class="heading mb-0"><b>Friday whole day (9am-5pm):</b></h2>
			  </br>
              <h3 class="heading mb-0">Software School for Phylogenomics</h3>
              <p><i>Primary organizer:</i> Dr. Tandy Warnow, University of Illinois Urbana-Champaign</br></br>
                 <i>Content:</i> This workshop provides hands-on training in numerous software tools for
				 phylogenetics and phylogenomics, including PASTA, UPP, ASTRAL, Phylonet, TIPP, and SVDquartets.
				 Participants will learn about methods for estimating large multiple sequence alignments and trees,
				 and how to estimate species trees in the presence of gene tree incongruence using genome-scale data.
				 This workshop was offered very successfully at a previous SSB Standalone (in 2015), and will be
				 updated to address new challenges and approaches for its 2020 offering.
			  </p>
             </div>

          <div class="text-left">
			  <h2 class="heading mb-0"><b>Monday morning (9am-12pm):</b></h2>
			  </br>
              <h3 class="heading mb-0">Broader Impacts and Science Communication </h3>
              <p><i>Primary organizer:</i> Sadie Mills, Florida Museum of Natural History</br></br>
                 <i>Content:</i> The goals of this workshop are to broaden participants’ perspectives
				 on conducting in-person and digital science communication, to help them craft and tailor
				 a message for both in-person and online delivery, and to educate participants on how to
				 best connect outreach to broader impact goals.
				 </br>
				 </br>
				 <i>Diversity statement:</i> The facilitators of this workshop are from multiple
				 organizations across the University of Florida campus, including staff from the
				 Florida Museum, iDigBio, and the Thompson Earth Systems Institute. Facilitators range
				 in career stage and educational and demographic backgrounds. 
			  </p>
          </div>
		  </br>
          <div class="text-left">
              <h3 class="heading mb-0">Phylogenic Comparative Analysis of Integrated Anatomical Traits </h3>
              <p><i>Primary organizer:</i> Dr. Josef Uyeda, Virginia Tech</br></br>
                 <i>Content:</i> Attendees will learn how to use R packages such as RPhenoscape
				 to access a knowledgebase of ontology-linked phenotypes, build character matrices
				 that take anatomical dependencies into account, and use these to construct stochastic
				 character maps on a phylogeny. The course will also include a practical introduction
				 to community ontologies for biodiversity domain knowledge (anatomy, taxonomy, phenotypic quality).
			  </p>
          </div>

          <div class="text-left">
			<h2 class="heading mb-0"><b>Monday afternoon (1:30-5pm):</b></h2>
			  </br>
              <h3 class="heading mb-0">Posterior Predictive Simulation with P2C2M</h3>
              <p><i>Primary organizer:</i> Dr. Bryan Carstens, Ohio State University</br></br>
                 <i>Content:</i> This workshop will introduce students to posterior predictive
				 simulation, an approach to evaluating the fit of statistical models used in a
				 Bayesian framework. In addition to the theoretical background, students will be
				 introduced to P2C2M, an R package that enables users to test the fit of the
				 multispecies coalescent to their sequence or SNP data.
			  </p>
          </div>


          <div class="text-left">
			  <h2 class="heading mb-0"><b>Monday whole day (9am-5pm): </b></h2>
			  </br>
              <h3 class="heading mb-0">Using Digitized Specimen Data in Research: Applications for Ecology, Phylogenetics, and Biogeography</h3>
              <p><i>Primary organizer:</i> Dr. Pamela Soltis, Florida Museum of Natural History</br></br>
                 <i>Content:</i> This workshop provides hands-on instructions on how to link and
				 analyze diverse data types from organismal biology, ecology, genetics, climatology,
				 and other disciplines, including information on phylogenetic diversity, species
				 distributions, changes in distributions over time, phenology, morphology, and more.
				 Participants will divide into groups based on experience. For beginners, we will cover
				 ways to access and download digitized herbarium data (from GBIF, iDigBio, and other
				 aggregators) and prepare data sets for analysis. We will then offer a series of modules
				 on using georeferencing software (GEOLocate) and applying Maxent software to construct
				 ecological niche models and do paleoclimatic modeling. For advanced users, we will provide
				 new, innovative modules for linking specimen data to phylogenetic trees, computing phylogenetic
				 diversity measures, conducting biogeographic analyses, and more. We will cover strategies to
				 extract information from niche models, such as species occupancy in ecological space and niche
				 breadth, and link them to phylogenetic trees to test hypotheses about niche evolution, such
				 as ancestral niche reconstruction. Participants will use new integrative software tools
				 developed by the BiotaPhy Project (www.biotaphy.org) in collaboration with the Lifemapper
				 Project that link occurrence data (through iDigBio), niche models, and ecological statistics
				 calculated from the models, applying these to large trees in a desktop geospatial environment
				 using the QGIS GIS application. Prepared datasets will be provided, but attendees may bring their own data.
			  </p>
          </div>
		  </br>
          <div class="text-left">
              <h3 class="heading mb-0">Analysis of Target Enrichment Data</h3>
              <p><i>Primary organizer:</i> Dr. Keith Barker, University of Minnesota</br></br>
                 <i>Content:</i> This workshop will focus on issues that can arise
				 when raw genomic data are assembled using different methods, with
				 resulting differences or biases in quality that can affect downstream
				 interpretation and analysis. The workshop will use available avian UCE
				 data and will walk participants through contig assembly, alignment, filtering,
				 and concatenation using Phyluce, providing all necessary code but with "precooked"
				 versions of each step available, especially where run times are prohibitive
				 (e.g., assemblies). Participants will then generate post assembly and alignment summaries
				 and conduct analyses of gene tree estimation (RAxML/MrBayes), species tree estimation
				 (Astral/SVDquartets), node assessment (via locus bootstrapping, ICA values in RAxML,
				 and gene and site metrics in IQ-tree), and possibly evaluation of hybridization hypotheses
				 (BUCKy and SNaQ) if time allows.
			  </p>
          </div>

      </div> 
	</div>
    <hr/>
    <div class="site-section">
      <div class="container">
         <div class="row">
          <div class="mx-auto text-center mb-5 section-heading">
            <h2 class="mb-5 text-uppercase">Discussion Forums</span> </h2>
			<p style = "font-size:1.5em"> Forum details TBD</p>
          </div> 
          </div> 

          <div class="text-center">
              <h3 class="heading mb-0">The State of Coalescent-based Methods in the Age of Big Data</h3>
              <p>Details TBD</p>
            </div>

          
          <div class="text-center">
              <h3 class="heading mb-0">The Potential of Machine Learning for Systematics</h3>
              <p>Details TBD</p>
            </div>

          <div class="text-center">

              <h3 class="heading mb-0">Cytonuclear Coevolution and Species Barriers</h3>
              <p>Details TBD</p>
            </div>

          <div class="text-center">
              <h3 class="heading mb-0">Combining Phylogenies and Ecological Data Layers</h3>
              <p>Details TBD</p>
            </div>
           
      </div>
    </div>




    
    <div class="py-5 quick-contact-info">
      <div class="container">
        <div class="row">
          <div class="col-md-4 text-center">
            <div>
              <span class="icon-room text-white h2 d-block"></span>
              <h2>Where</h2>
              <p class="mb-0">Gainesville, FL</p>
            </div>
          </div>
          <div class="col-md-4 text-center">
            <div>
              <span class="icon-clock-o text-white h2 d-block"></span>
              <h2>When</h2>
              <p class="mb-0">January 3-6th, 2020</p>
            </div>
          </div>
          <div class="col-md-4 text-center">
            <div>
              <span class="icon-comments text-white h2 d-block"></span>
              <h2>Email</h2>
              <p class="mb-0">Email: Emily Sessa <br/><a href=mailto:ssb2020UF@gmail.com>ssb2020UF@gmail.com</a></p>
            </div>
          </div>
        </div>
      </div>
    </div>

    
    <footer class="site-footer" style="background-image: url('images/MWalters_5.jpg');">
      <div class="container">
        

        <div class="row">
          <div class="col-md-5">
            <h3 class="footer-heading mb-4 text-white">About SSB</h3>
			<p>The Society of Systematic Biologists advances the science of systematic biology
			in all its aspects of theory, principles, methodology, and practice, for both living
			and fossil organisms, with emphasis on areas of common interest to all systematic biologists
			regardless of individual specialization.</p>
            <p><a href="https://www.systbio.org/" class="btn btn-primary pill text-white px-4">Read More</a></p>
          </div>
          <div class="col-md-4">
            <div class="row">
              <div class="col-md-6">
                <h3 class="footer-heading mb-4 text-white">Quick Menu</h3>
                  <ul class="list-unstyled">
                      <li><a href="index.html">Home</a></li>
                      <li><a href="registration.html">Registration & Abstracts</a></li>
					  <li><a href="schedule.html">Schedule</a></li>
                      <li><a href="location.html">Location</a></li>
                      <li><a href="travel_accomodation.html">Travel & Accommodations</a></li>
                      <li><a href="workshop.html">Workshops & Forums</a></li>
					  <li><a href="activites.html">Activities</a></li>
                      <li><a href="contact.html">Contact</a></li>
					  <li><a href="acknowledgments.html">Acknowledgments</a></li>
                  </ul>
              </div>
            </div>
          </div>

          
          <div class="col-md-2">
            <div class="col-md-12"><h3 class="footer-heading mb-4 text-white">Social Icons</h3></div>
              <div class="col-md-12">
                <p>
                  <a href="https://twitter.com/hashtag/SSB2020?src=hash" class="p-2" style="font-size:2rem;"><span class="icon-twitter"></span></a>
                  <a href="https://www.facebook.com/groups/SocietySystematicBiologists/" class="pb-2 pr-2 pl-0" style="font-size:2rem;"><span class="icon-facebook"></span></a>
                </p>
              </div>
          </div>
        </div>
        <div class="row pt-5 mt-5 text-center">
          <div class="col-md-12">
            <p>
            <!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
            Copyright &copy; <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>document.write(new Date().getFullYear());</script> All Rights Reserved | This template is made with <i class="icon-heart text-primary" aria-hidden="true"></i> by <a href="https://colorlib.com" target="_blank" >Colorlib</a>
            <!-- Link back to Colorlib can't be removed. Template is licensed under CC BY 3.0. -->
            </p>
          </div>
          
        </div>
      </div>
    </footer>
  </div>

  <script src="js/jquery-3.3.1.min.js"></script>
  <script src="js/jquery-migrate-3.0.1.min.js"></script>
  <script src="js/jquery-ui.js"></script>
  <script src="js/popper.min.js"></script>
  <script src="js/bootstrap.min.js"></script>
  <script src="js/owl.carousel.min.js"></script>
  <script src="js/jquery.stellar.min.js"></script>
  <script src="js/jquery.countdown.min.js"></script>
  <script src="js/jquery.magnific-popup.min.js"></script>
  <script src="js/bootstrap-datepicker.min.js"></script>
  <script src="js/aos.js"></script>

  
  <script src="js/mediaelement-and-player.min.js"></script>

  <script src="js/main.js"></script>
    

  <script>
      document.addEventListener('DOMContentLoaded', function() {
                var mediaElements = document.querySelectorAll('video, audio'), total = mediaElements.length;

                for (var i = 0; i < total; i++) {
                    new MediaElementPlayer(mediaElements[i], {
                        pluginPath: 'https://cdn.jsdelivr.net/npm/mediaelement@4.2.7/build/',
                        shimScriptAccess: 'always',
                        success: function () {
                            var target = document.body.querySelectorAll('.player'), targetTotal = target.length;
                            for (var j = 0; j < targetTotal; j++) {
                                target[j].style.visibility = 'visible';
                            }
                  }
                });
                }
            });
    </script>

  </body>
</html>
